### ISSUE TITLE
Allow option to disable rendering "<!-- STORY -->" for README-only purposes

### ISSUE THREAD

**Issue by silentHoo** [tuchk4/storybook-readme#54]
First of all, thank you for the nice addon. It would be nice if there's an option to remove the story completely from the output. This is simply a feature request :)

**Comment by tuchk4** [comment]
There is PR with feature implementation.
Current API to do this is:

```js
import { doc } from 'storybook-readme';
import CommonDocs from './CommonDocs.md';

storiesOf('Doc', module)
  .add('Common', 
      doc(CommonDocs)
  );
``

**Comment by silentHoo** [comment]
Yeah, very nice 👍

But there's a little mistake in the sample code. It should be:

`() => doc(CommonDocs)` instead of
`doc(CommonDocs)`

**Comment by tuchk4** [comment]
maybe should left it like this `() => doc(CommonDocs)` so codebase will be lesser then now :)

**Comment by silentHoo** [comment]
Sorry, I responded too fast. I played around myself in the sources of `storybook-readme` and I didn't discard my local changes. So the `doc()` method won't work with the current release. Sorry my fault :)

**Comment by stevebertan** [comment]
Here is how I've implemented it until the new API is released:

```js
import styled from 'styled-components';
import { withDocs } from 'storybook-readme';
import CommonDocs from './CommonDocs.md';

const doc = withDocs({
  PreviewComponent: styled.div`
    display: none;
  `,
});

storiesOf('Doc', module)
  .add('Common', 
      doc(CommonDocs)
  );
```

**Comment by tuchk4** [comment]
new api released at `storybook-readme@3.3.0`
